### PR TITLE
[plugin.video.cbc] 4.0.10+matrix.1

### PR DESCRIPTION
**Pull request by micahg**
### Description
* Add live streams to IPTV Manager
* Add EPG data to IPTV Manager
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.cbc/resources/lib/utils.py [hunk 1]
```diff
@@ -85,5 +93,6 @@ def log(msg, error = False):
         import xbmc
         full_msg = "plugin.video.cbc: {}".format(msg)
         xbmc.log(full_msg, level=xbmc.LOGERROR if error else xbmc.LOGINFO)
+        print(xbmc.log)
```
Comment (enen92) [inline plugin.video.cbc/resources/lib/utils.py]: Leftover?

**Review by enen92 — COMMENTED** [review]
See comment

File: plugin.video.cbc/resources/lib/utils.py (hunk 1 above)
Comment (micahg) [reply plugin.video.cbc/resources/lib/utils.py]: Yes :(

I'll repush

Comment (micahg) [reply plugin.video.cbc/resources/lib/utils.py]: @enen92 done

**Review by enen92 — APPROVED** [review]

**Pull request merged**